tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -623,3 +632,32 @@ def validate_header_rpt_month_year(datafile, header, generate_error):
             field=None,
         )
     return is_valid, error
+
+
+def t3_child_validator(which_child):
+    """T3 child validator."""
+    def t3_first_child_validator_func(value):
+        if not _is_empty(value, 1, 60) and len(value) >= 60:
+            return (True, None)
+        else:
+            return (False, "1st child record truncated.")
```

Comment:
nit. Can we use the words First and Second instead of 1st?

Reply:
This is what Alex wanted.

Reply:
Copy that